chaos-tests/Readme.md

```diff
@@ -0,0 +1,42 @@
+# Chaos Mesh tests for Kubernetes Client SharedInformer
+
+This module will run automated Chaos Tests for the SharedInformers
```

Comment:
It would be also nice to have a summary here of what the test is doing:
- what in this ChaosMesh is doing ?
- high level description of what and how the Checker and Control are working (explain the counter story?)

Reply:
Thanks this is just perfect, I think it is worth adding this in the Readme or in ChaosIT javadoc. I agree this is kind of a translation in English but at that stage, the whole logic is located in several places which makes the story a bit hard to follow.

Reply:
@sunix added something to the Readme, let me know if you think it's enough.

Reply:
Have you push your changes?

Reply:
ooops, wrong branch 🤦 done now!

Reply:
Perfect thanks !!!